test/test_nn.py

```diff
@@ -3233,6 +3233,13 @@ def func(root):
         gradcheck(func, [v])
         gradgradcheck(func, [v])
 
+    def test_batchnorm_double_backward(self):
+        bn = nn.BatchNorm1d(5).cuda()
```

Comment:
The CUDA calls should be guarded by a check or a `@unittest.skipIf`.

Are we doing gradgradchecks of BatchNorm? For both THNN and cuDNN?

Reply:
Good catch, I forgot to reenable the tests I disabled >.>